<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="1" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-01T15:25:16 IST" name="Default suite" finished-at="2024-03-01T15:25:27 IST" duration-ms="10180">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidationsTest.LoginError()[pri:0, instance:rahulshettyAcademy.tests.ErrorValidationsTest@3ba987b8]" name="LoginError" class="rahulshettyAcademy.tests.ErrorValidationsTest"/>
      </group> <!-- ErrorHandling -->
    </groups>
    <test started-at="2024-03-01T15:25:16 IST" name="Default test" finished-at="2024-03-01T15:25:27 IST" duration-ms="10180">
      <class name="rahulshettyAcademy.tests.ErrorValidationsTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshettyAcademy.tests.ErrorValidationsTest@3ba987b8]" started-at="2024-03-01T15:25:17 IST" name="launchApplication" finished-at="2024-03-01T15:25:23 IST" duration-ms="6095" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="LoginError()[pri:0, instance:rahulshettyAcademy.tests.ErrorValidationsTest@3ba987b8]" started-at="2024-03-01T15:25:23 IST" name="LoginError" finished-at="2024-03-01T15:25:27 IST" duration-ms="3925" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginError -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshettyAcademy.tests.ErrorValidationsTest@3ba987b8]" started-at="2024-03-01T15:25:27 IST" name="tearDown" finished-at="2024-03-01T15:25:27 IST" duration-ms="108" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- rahulshettyAcademy.tests.ErrorValidationsTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
